api-reference/10 UI Components/dxScrollable/1 Configuration/useNative.md

```diff
@@ -5,6 +5,10 @@ default: true, false (desktop except Mac)
 ---
 ---
 ##### shortDescription
-Indicates whether to use native or simulated scrolling.
+Specifies whether the component uses native or simulated scrolling.
 
 ---
+
+When this property is set to `true`, ScrollView implements the scrolling logic of the browser. When disabled, the component implements custom scrolling logic that does not change across browsers and devices.
```

Comment:
``When this property is set to `true`, ScrollView implements the scrolling logic of the browser.``

If ScrollView uses the browser's scrolling logic, then is any logic actually _implemented_? For example, would it be better to say:

``When this property is set to `true`, ScrollView uses the browser's scrolling logic.``